.gitignore

```diff
@@ -1,5 +1,6 @@
 site/
 .venv/
+__pycache__/
```

Comment:
**question:** This seems to be an unrelated change. What's the motivation or where does it come from?

Reply:
It get created automatically when I was running `make serve`, but with `mkdocs` coming from `.venv`.

Reply:
I guess it's fine to add this to `.gitignore`. But let's drop it here and do a separate PR for that.
/cc @oprypin